src/backend/apps/rest/tasks.py

```diff
@@ -32,8 +32,18 @@ def populate_all_rest_stop_data():
     client = FeedClient()
     feed_data = client.get_rest_stop_list()
 
+    # Purge rest stops absent in the feed
+    available_rest_stop_ids = []
     for rest_stop_data in feed_data:
-        populate_rest_stop_from_data(rest_stop_data)
-
+        try:
+            chris_rest_area_id = rest_stop_data.get('properties', {}).get('CHRIS_REST_AREA_ID')
+            populate_rest_stop_from_data(rest_stop_data)
+            if chris_rest_area_id:
+                available_rest_stop_ids.append(chris_rest_area_id)
+        except Exception as e:
+            logger.warning(e)
```

Comment:
What exception are we expecting here?

Reply:
Adding all the available rest stop ids from the API to the list that will be used for the deletion from the database.